scripts/check-websockets.ts

```diff
@@ -84,10 +84,12 @@ async function checkWebSocketEndpoints(endpoints: ChainEndpoint[]): Promise<{
   return { failingEndpoints, successfulEndpoints };
 }
 
-function filterChainList(includeTestChains: boolean): AnyChain[] {
-  return includeTestChains
-    ? chainsList
-    : chainsList.filter((chain) => !chain.isTestChain);
+function filterChainList(includeTestChains: boolean): AnyParachain[] {
+  return (
+    includeTestChains
+      ? chainsList
+      : chainsList.filter((chain) => !chain.isTestChain)
+  ).filter((chain) => Parachain.is(chain) || EvmParachain.is(chain));
```

Comment:
This doesn't seems like a biome change, although the change itself makes sense. I'm just pointing it out in case it was accidental

Reply:
It was not an accident. I just saw that the types were wrong. Just `Parachain` and `EvmParachain` have an `ws` endpoint. So I decided to fix it.